.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+_**Please edit this to reflect the pull request. A pullr equest with this line in it will not(!!!) be accepted.**_
+### Description
+A short explaination of the pullrequest if needed. could be "we added support for Python" or a long ass text. idealy a few lines.
+
+### Changelog
+ - this has changed
+ - this has aswell 
+   - which includes this
+   - and also this
+
+### Issues solved 
+add these with the respective key as HCLANG-1 (as an example)
+ - HCLANG-[number]
+
+### Issues opened as a result of this 
+Add something here if new issues arrise because of this. For instance if new features-needs or bugs arrise from this PR. if this is not the cause, then remove this section
+
+## FOR REVIEWER [Do not remove]
+A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
+ - Did the PR actually fix the issue
+ - Comment anything that doesn't make sense or anything you want clarified. Many comments are always excuseable
+ - Make sure this is ready for exams if this is going into master
+ - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.
+
+### Regarding code [if applicable]
+- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
```

Comment:
I think CI's should ensure that code conventions are being followed

Reply:
it's always good to have a second pair of eyes on the code i'd say. like ideally yes but it's nice to have an extra pair of eyes making sure that it not complete garbish